### PR TITLE
Parsigs universe

### PR DESCRIPTION
**Pull request by royashcenazi**
## Description
Add Parsigs project to the Universe section.

### Types of change
Enhancement

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -1,5 +1,27 @@
 {
     "resources": [
+        {
+            "id": "parsigs",
+            "title": "parsigs",
+            "slogan": "Structuring prescriptions text made simple using Spacy",
```
Comment (bdura) [inline website/meta/universe.json:6]:
```suggestion
            "slogan": "Structuring prescriptions text made simple using spaCy",
```

File: website/meta/universe.json [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+            "description": "Parsigs is an open-source project that aims to extract the relevant dosage information from prescriptions text without compromising the patient׳s privacy.",
```
Comment (bdura) [inline website/meta/universe.json:7]:
```suggestion
            "description": "Parsigs is an open-source project that aims to extract the relevant dosage information from prescriptions text without compromising the patient's privacy.",
```

**Review by bdura — CHANGES_REQUESTED** [review]
Hello @royashcenazi , thanks for the submission!

I took it for a test drive, right now the example fails because of the pre-trained model dependency. Could you maybe add it to your requirements file? I'd also suggest adding version constraints to spaCy (eg spacy>=3.0,<4.0), otherwise you might get compatibility issues in the future.

I've also made minor suggestions to the description and tagline.

**Comment by royashcenazi** [comment]
> Hello @royashcenazi , thanks for the submission!
> 
> I took it for a test drive, right now the example fails because of the pre-trained model dependency. Could you maybe add it to your requirements file? I'd also suggest adding version constraints to spaCy (eg spacy>=3.0,<4.0), otherwise you might get compatibility issues in the future.
> 
> I've also made minor suggestions to the description and tagline.

**Pull request closed without merging**